src/org/labkey/targetedms/SkylinePort/Irt/RegressionLineElement.java

```diff
@@ -59,7 +59,8 @@ public double getIntercept()
 		return _regressionLine.getIntercept();
 	}
 
-	public double GetY(double x)
+	@Override
+    public double GetY(double x)
```

Comment:
Formatting looks mangled. Maybe it was pre-mangled and we should do a general reformat in this PR too?